README.md

```diff
@@ -1,24 +1,36 @@
 # Freenas-Quicksync
-###### How to guide for getting Intel Quicksync working on Freenas
-While researching this, everything I found led me to believe that Intel Quicksync would not work until we got FreeBSD 12. After researching forums online, I have gotten it to work on 11.3. I am not much more than a noob that is only tinkering but this works for me. I am using a i7 4790K and using Quicksync for 1 1080p to 720p or 1080p to 1080p stream leaves my cpu usage at 1%. I have not tested much more yet.
+##### How to guide for getting Intel Quicksync working on Freenas
+While researching this, everything I found led me to believe that Intel Quicksync would not work until we got FreeBSD 12. After researching forums online, I have gotten it to work on 11.3. I am not much more than a noob that is only tinkering but this works for me.
+## Test Results
+- I am using a i7 4790K and using Quicksync for 1 1080p to 720p or 1080p to 1080p stream leaves my cpu usage at 1%.
+- Core 4th Gen (Haswell) added H.265 and VC1 support in QuickSync. For 1080p transcode, "Haswell or better" is reasonable guidance.
+- Testing on a 1225v6 Xeon (7th Gen Kaby Lake) used ~4% CPU for a 4k HDR -> 1080p SDR hardware transcode. The same machine was at 100% CPU for a software transcode of 4k -> 1080p. Running four 4k hardware transcodes at once did not significantly tax the machine, further testing to determine the max number of possible 4k transcodes was not done.
+- Testing on a G4560 Pentium (7th Gen Kaby Lake) used ~30% CPU for a 4k -> 1080p transcode.
+- Testing on a Core i3 6100T (6th Gen SkyLake) shows that acceleration of a 4k -> 1080p transcode is minimal on it and takes 95% CPU. This is likely due to limited HEVC features in that generation.
```

Comment:
The format that's being transcoded is more important than the resolution.
What is currently writhen here is not really relevant, because important info is missing

Reply:
Done. I don't know what codecs was used in the original 1080p -> 720p test, so I've not added that information.